service/src/main/java/com/epam/rft/atsy/service/impl/ChannelServiceImpl.java

```diff
@@ -40,6 +40,9 @@
     @Override
     public void saveOrUpdate(ChannelDTO channel) {
         Assert.notNull(channel);
+        Assert.notNull(channel.getId());
```

Comment:
The new channels which should be created have no id-s, so this assertion will prevent inserting new channels.

Please see the failing integration tests.